tests/build_/test_build_request.py

```diff
@@ -1469,6 +1468,7 @@ def test_render_optional_plugins(self, tmpdir):
             'kojihub': 'kojihub',
             'kojiroot': 'kojiroot',
             'pulp_registry': 'pulp_registry',
+            'pulp_secret': 'pulp_secret',
```

Comment:
This is odd... Why is it needed now? Seems to be that pulp_secret shouldn't be passed along to worker builds.

Reply:
It's needed for the orchestrator build to render pulp_tag, apparently.

Reply:
Was this not needed after all?